server/src/domain/person/person.service.ts

```diff
@@ -345,7 +345,9 @@ export class PersonService {
     } as const;
 
     await this.mediaRepository.resize(croppedOutput, thumbnailPath, thumbnailOptions);
-    await this.repository.update({ id: personId, thumbnailPath });
+    if (personId) {
```

Comment:
How can person id ever not be defined/available? 

This is called with the _id_ of a person.
The job returns if the person doesn't have a face

Under what situations will personId be null and should not be returned? If this is just a type checking issue, maybe you should update it to use `person.id` instead.

Reply:
> Under what situations will personId be null and should not be returned?

Indeed, that situation does not exist.